tests/core/test_invite.py

```diff
@@ -111,7 +113,7 @@ async def _run_greeter():
 
         await in_progress_ctx.do_create_new_device(author=alice, device_label=granted_device_label)
 
-    with trio.fail_after(1):
+    with trio.fail_after(3):
```

Comment:
should not be needed, (well the correct way of doing this is to replace this by the new shinny `real_clock_timeout` which provide a consistent across tests (and configurable) timeout time

Reply:
Changed to real_clock_timeout